charts/private-action-runner/README.md

```diff
@@ -0,0 +1,54 @@
+    * Replace the `URN_FROM_CONFIG` and the `PRIVATE_KEY_FROM_CONFIG` in the example file with with the `urn` and the `privateKey` from the `config/config.yaml` of the docker container.
+    * You can reconfigure other values or use the defaults in the example.
+6. Add this repository to your Helm repositories:
+    ```
+    helm repo add datadog https://helm.datadoghq.com
+    helm repo update
+    ```
+7. Install the Helm chart:
+    ```bash
+        helm install <RELEASE_NAME> datadog/private-action-runner -f ./config.yaml
+    ```
+8. Go to the [Workflow connections page](https://app.datadoghq.com/workflow/connections).
+9. Create a new connection, select your private action runner, and use **Service account authentication**.
+10. Create a new workflow and use a Kubernetes action like **List pod** or **List deployment**.
+
+## Going further
+* Adjust the service account permissions according to your needs. Learn more about [Kubernetes RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac).
+* Deploy several runners with different permissions or create different connections according to your needs.
+* Learn more about [Private actions](https://docs.datadoghq.com/service_management/app_builder/private_actions).
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| common.image | string | `"us-east4-docker.pkg.dev/datadog-sandbox/apps-on-prem/onprem-runner:v0.0.1-alpha23"` | Current Datadog Private Action Runner image |
+| runners[0].config | object | `{"actionsAllowlist":["com.datadoghq.kubernetes.core.listPod"],"appBuilder":{"port":9016},"ddBaseURL":"https://app.datadoghq.com","modes":["workflowAutomation","appBuilder"],"privateKey":"PRIVATE_KEY_FROM_CONFIG","urn":"URN_FROM_CONFIG"}` | Configuration for the Datadog Private Action Runner |
+| runners[0].config.actionsAllowlist | list | `["com.datadoghq.kubernetes.core.listPod"]` | List of actions that the Datadog Private Action Runner is allowed to execute |
+| runners[0].config.appBuilder.port | int | `9016` | Required port for App Builder Mode |
+| runners[0].config.ddBaseURL | string | `"https://app.datadoghq.com"` | Base URL of the Datadog app |
+| runners[0].config.modes | list | `["workflowAutomation","appBuilder"]` | Modes that the runner can run in |
+| runners[0].config.privateKey | string | `"PRIVATE_KEY_FROM_CONFIG"` | The runner's privateKey from the enrollment page |
+| runners[0].config.urn | string | `"URN_FROM_CONFIG"` | The runner's URN from the enrollment page |
+| runners[0].kubernetesPermissions | list | `[{"apiGroups":[""],"resources":["pods"],"verbs":["list","get"]},{"apiGroups":["apps"],"resources":["deployments"],"verbs":["list","get"]}]` | List of Kubernetes permissions that the Datadog Private Action Runner has |
+| runners[0].name | string | `"default"` | Name of the Datadog Private Action Runner |
+| runners[0].replicas | int | `1` | Number of pod instances for the Datadog Private Action Runner |
```

Comment:
```suggestion
| Key                                | Description                                                                  | Type   | Default                                                                                                                                                                                                                                        |
|------------------------------------|------------------------------------------------------------------------------| -------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| common.image                       | Current Datadog Private Action Runner image                                  | string | `"us-east4-docker.pkg.dev/datadog-sandbox/apps-on-prem/onprem-runner:v0.0.1-alpha23"`                                                                                                                                                          |
| runners[0].config                  | Configuration for the Datadog Private Action Runner                          | object | `{"actionsAllowlist":["com.datadoghq.kubernetes.core.listPod"],"appBuilder":{"port":9016},"ddBaseURL":"https://app.datadoghq.com","modes":["workflowAutomation","appBuilder"],"privateKey":"PRIVATE_KEY_FROM_CONFIG","urn":"URN_FROM_CONFIG"}` |
| runners[0].config.actionsAllowlist | List of actions that the Datadog Private Action Runner is allowed to execute | list   | `["com.datadoghq.kubernetes.core.listPod"]`                                                                                                                                                                                                    |
| runners[0].config.appBuilder.port  | Required port for App Builder Mode                                           | int    | `9016`                                                                                                                                                                                                                                         |
| runners[0].config.ddBaseURL        | Base URL of the Datadog app                                                  | string | `"https://app.datadoghq.com"`                                                                                                                                                                                                                  |
| runners[0].config.modes            | Modes that the runner can run in                                             | list   | `["workflowAutomation","appBuilder"]`                                                                                                                                                                                                          |
| runners[0].config.privateKey       | The runner's privateKey from the enrollment page                             | string | `"PRIVATE_KEY_FROM_CONFIG"`                                                                                                                                                                                                                    |
| runners[0].config.urn              | The runner's URN from the enrollment page                                    | string | `"URN_FROM_CONFIG"`                                                                                                                                                                                                                            |
| runners[0].kubernetesPermissions   | List of Kubernetes permissions that the Datadog Private Action Runner has    | list   | `[{"apiGroups":[""],"resources":["pods"],"verbs":["list","get"]},{"apiGroups":["apps"],"resources":["deployments"],"verbs":["list","get"]}]`                                                                                                   |
| runners[0].name                    | Name of the Datadog Private Action Runner                                    | string | `"default"`                                                                                                                                                                                                                                    |
| runners[0].replicas                | Number of pod instances for the Datadog Private Action Runner                | int    | `1`                                                                                                                                                                                                                                            |
```

Reply:
@neko-dd this section of the README is auto-generated from the comments in values.yaml. If I change it directly then it will cause tests to fail. It looks like these are formatting changes? so I don't think I am able to change this

Reply:
I see. There's no way to reorder the columns, I take it? That's mostly what I was trying to do.

Reply:
yeah I don't think so unfortunately

Reply:
Okay. Alas!